src/pythermalcomfort/jos3_functions/construction.py

```diff
@@ -12,11 +12,24 @@
 """
 
 import numpy as np
+from dataclasses import dataclass
 from pythermalcomfort.jos3_functions.matrix import NUM_NODES, IDICT, BODY_NAMES
 from pythermalcomfort.utilities import body_surface_area
 
-# Body surface area of the standard body [m2]
-_BSAst = np.array(
+
+# Anthropomorphic data for a default body
+@dataclass
+class Default:
+     height: float = 1.72
+     weight: float = 74.43
+     age: int = 20
+     body_fat: float = 15
+     cardiac_index: float = 2.59  # [L/min/m2]
+     sex: str = "male"
+     posture: str = "standing"
+     bmr_equation: str = "harris-benedict"
+     bsa_equation: str = "dubois"
+     local_bsa: float = np.array( # body surface area [m2]
```

Comment:
I think you may have selected the wrong data type here since you are saying that local_bsa is a float while instead is an array of floats. Could you please check this?

Reply:
Oh sorry, and thank you for pointing this out! I have changed it to np.ndarray from float.